src/indicators/flooding/glofas/get_stations.py

```diff
@@ -0,0 +1,38 @@
+"""
+Script to turn csv of GloFAS stations into config format
+for aa toolbox. Input is country ISO2 code.
+"""
+import os
+import sys
+from pathlib import Path
+
+import pandas as pd
+
+DATA_DIR = Path(os.environ["AA_DATA_DIR"])
+FILENAME = (
+    DATA_DIR
+    / "private/exploration/glb/glofas/Qgis_World_outlet_202104_20210421.csv"
```

Comment:
what is the source of the file? How did you get it and what does it contain? Should it be updated from time to time?